spec/features/projects_manage_spec.rb

```diff
@@ -427,9 +432,39 @@
     end
   end
 
+  describe "when unlocking a project" do
+    context "when a user is an admin" do
+      let(:user) { FactoryBot.create(:user, admin: true) }
+
+      context "if a project is locked" do
+        let(:locked_project) { FactoryBot.create(:project, :locked) }
+
+        it "shows an unlocked button" do
+          visit project_path(locked_project)
+          within "div.btn-group.actions" do
+            expect(page).to have_text("Unlock Project")
+          end
+        end
+
+        it "unlocks a project" do
```

Comment:
Perhaps we can be a bit more descriptive about the action that leads to the project being unlocked?
```suggestion
        it "unlocks the project when 'Unlock Project' is clicked" do
```